Moving to robot-snapshots (#10)
icub-tech-iit/icub-snapshots: README.md

Robot Snapshots
===============

> This repository contains the CAD snapshots for each robot version, S/N and/or batch of robots.

Author: ```suggestion
This repository contains the snapshots for each robot version, S/N and/or batch of robots.
```

Actually, it's more generic. Let me restore the original version.

For each robot snapshot we created a dedicated `git branch`.

As an example, if you want to open the shapshot of the robot version `2.6`, switch to its dedicated branch [`iCub2_6`](https://github.com/icub-tech-iit/icub-snapshots/tree/iCub2_6).

If you own an iCub robot, you can find the snapshot of your robot S/N in the branch named `icub_sn_YOUR-ROBOT-SERIAL-NUMBER`.
**NOTE**: if your robot S/N is not present, refer to the snapshot of the associated robot version.

### 🔽 How to clone specific branches locally
```sh
$ git clone https://github.com/icub-tech-iit/robot-snapshots.git --single-branch --branch <branch-name>
```

⚠ Be careful that branches are handled via [Git LFS](https://help.github.com/en/articles/installing-git-large-file-storage).

## Maintainers

> This repository is maintained by the CAD designer in the Team PROTO.

Author: ```suggestion
The CAD designer of Team PROTO maintains this repository.
```

> This repository is maintained by the CAD designer in the Team PROTO.

Author: As this repo is public, links to internal resources shall be avoided.
